Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/TaskFactory.kt

```diff
@@ -0,0 +1,20 @@
+package de.rki.coronawarnapp.task
+
+interface TaskFactory<
```

Comment:
There are a lot of Task_Sth_ classes in the task package - some of which are interfaces, others are implementation, models, ... 
How about introducing an "... .task.internal" package to host the implementation details and all public interfaces remain in task